CHANGELOG.md

```diff
@@ -12,11 +12,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#505](https://github.com/nf-core/taxprofiler/pull/505) - Add small files to the file `tower.yml` (added by @LilyAnderssonLee)
 - [#508](https://github.com/nf-core/taxprofiler/pull/508) - Add `nanoq` as a filtering tool for nanopore reads (added by @LilyAnderssonLee)
 - [#511](https://github.com/nf-core/taxprofiler/pull/511) - Add `porechop_abi` as an alternative adapter removal tool for long reads nanopore data (added by @LilyAnderssonLee)
+- [#512](https://github.com/nf-core/taxprofiler/pull/512) - Update all tools to the latest version and include nf-test (Updated by @LilyAnderssonLee & @jfy133)
 
 ### `Fixed`
 
+- [#518](https://github.com/nf-core/taxprofiler/pull/518) Fixed a bug where Oxford Nanopore FASTA input files would not be processed (❤️ to @ikarls for reporting, fixed by @jfy133)
+
 ### `Dependencies`
 
+| Tool                     | Previous version | New version |
+| ------------------------ | ---------------- | ----------- |
+| bbmap                    | 39.01            | 39.06       |
+| bowtie2                  | 2.4.4            | 2.5.2       |
+| bracken                  | 2.7              | 2.9         |
+| cat/fastq:coreutils=8.30 |                  | 8.30        |
```

Comment:
What about 

```suggestion
| cat/fastq |                  | 8.30        |
```

Reply:
I am not sure. Since coreutils=8.30 was used in the module. If you think it's proper, I could update.

Reply:
Maybe just `cat/fastq` is enough?